### ISSUE TITLE
Unsatisfiable requirements detected for package MbedTLS

### ISSUE THREAD

**Issue by Thibaut-Deveraux** [GenieFramework/Genie.jl#181]
Hi,

I got this when trying the tutorial. I will try on other platefroms.

**Describe the bug**

On Ubuntu 18.04.3 LTS, I just did:

```
pkg> add DataFrame 
pkg> add CSV
pkg> add Genie
julia>using Genie
julia> Genie.newapp("MyGenieApp")
```
The last line throw an error.

**Error stacktrace**
```
ERROR: Unsatisfiable requirements detected for package MbedTLS [739be429]:
 MbedTLS [739be429] log:
 ├─possible versions are: [0.5.13-0.5.14, 0.6.0-0.6.8, 0.7.0] or uninstalled
 ├─restricted to versions 0.7.0-0.7 by Genie [c43c736e], leaving only versions 0.7.0
 │ └─Genie [c43c736e] log:
 │   ├─possible versions are: 0.19.2 or uninstalled
 │   └─Genie [c43c736e] is fixed to version 0.19.2
 └─restricted by julia compatibility requirements to versions: [0.5.13-0.5.14, 0.6.0-0.6.8] or uninstalled — no versions left
Stacktrace:
 [1] #propagate_constraints!#61(::Bool, ::Function, ::Pkg.GraphType.Graph, ::Set{Int32}) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1007
 [2] propagate_constraints! at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:948 [inlined]
 [3] #simplify_graph!#121(::Bool, ::Function, ::Pkg.GraphType.Graph, ::Set{Int32}) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1462
 [4] simplify_graph! at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:1462 [inlined] (repeats 2 times)
 [5] resolve_versions!(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Nothing) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:373
 [6] resolve_versions! at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:316 [inlined]
 [7] #add_or_develop#62(::Array{Base.UUID,1}, ::Symbol, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1201
 [8] #add_or_develop at ./none:0 [inlined]
 [9] #add_or_develop#15(::Symbol, ::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69
 [10] (::getfield(Pkg.API, Symbol("#kw##add_or_develop")))(::NamedTuple{(:mode,),Tuple{Symbol}}, ::typeof(Pkg.API.add_or_develop), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at ./none:0
 [11] do_add!(::Dict{Symbol,Any}, ::Array{Pkg.Types.PackageSpec,1}, ::Dict{Symbol,Any}) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/REPLMode.jl:709
 [12] #invokelatest#1(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Any, ::Any, ::Vararg{Any,N} where N) at ./essentials.jl:697
 [13] invokelatest(::Any, ::Any, ::Vararg{Any,N} where N) at ./essentials.jl:696
 [14] do_cmd!(::Pkg.REPLMode.PkgCommand, ::Pkg.REPLMode.MiniREPL) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/REPLMode.jl:603
 [15] #do_cmd#33(::Bool, ::Function, ::Pkg.REPLMode.MiniREPL, ::String) at /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.0/Pkg/src/REPLMode.jl:577
 [16] #do_cmd at ./sysimg.jl:0 [inlined]
 [17] install_app_dependencies(::String) at /home/timilie/.julia/packages/Genie/1HgKL/src/Generator.jl:328
 [18] #newapp#24(::Bool, ::Bool, ::Bool, ::Bool, ::Function, ::String) at /home/timilie/.julia/packages/Genie/1HgKL/src/Generator.jl:437
 [19] newapp(::String) at /home/timilie/.julia/packages/Genie/1HgKL/src/Generator.jl:418
 [20] top-level scope at none:0
```

**Behavior**

- The files were created.
- It did cd into MyGenieApp
- It seem to have activated the project in pkg.
- localhost:8000 return ERR_CONNECTION_REFUSED. ufw firewall is inactive.

**Additional context**

```
julia> versioninfo()
Julia Version 1.0.4
Commit 38e9fb7f80 (2019-05-16 03:38 UTC)
Platform Info:
  OS: Linux (i686-pc-linux-gnu)
  CPU: Intel(R) Pentium(R) 3558U @ 1.70GHz
  WORD_SIZE: 32
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, haswell)
```

```
(MyGenieApp) pkg> st
    Status `~/code/genie-test/MyGenieApp/MyGenieApp/Project.toml`
  (empty environment)
```

**Comment by Thibaut-Deveraux** [comment]
No problem on Windows with Julia 1.2. I may have an issue on my side...

```
Julia Version 1.2.0
Commit c6da87ff4b (2019-08-20 00:03 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_REVISE = auto
```

```
(MyGenieApp) pkg> st
    Status `C:\Users\Shadow\AppData\Local\Julia-1.2.0\MyGenieApp\Project.toml`
  [c43c736e] Genie v0.19.2 #master (https://github.com/GenieFramework/Genie.jl.git)
  [e6f89c97] LoggingExtras v0.3.0
  [295af30f] Revise v2.2.2
```

**Comment by Thibaut-Deveraux** [comment]
Solved by updating Julia to 1.2 on Linux too. Using Generic Linux Binaries of v1.2.0 instead of the LTS v1.0.5 from Snap.

**Comment by essenciary** [comment]
@Thibaut-Deveraux Thanks for letting me know. Looks like a dependency versions issue. Glad you got it sorted out.
